### DISCUSSION TITLE
Scoop can't install "Neofetch", even with Git insatlled

### DISCUSSION THREAD

**Discussion by VictiniFan360** [ScoopInstaller/Scoop#5412] [Help and Questions]
Helllo!
In my Windows 10 I want to install Vanilla Neofetch using this package manager.
I could install Scoop succesfully yesterday. However, when I wrote in PowerShell "scoop install neofetch", it return me an error message saying that "git" is not installed.
I installed "git", but when I put Neofetch's installation command I kept reciving this error.
Nonetheless, today I started PowerShell and it could install this software succesfully, but the terminal doesn't "recognize" it, but the file "neofetch" appear in my user's folder.
AFAIK, I installed Windows 10 with a local account, but then I used my Microsoft Account.
![Scoop error 1](https://user-images.githubusercontent.com/70706948/221694195-05958373-307d-499e-ba5e-083889831aa2.png)
![Scoop error 2](https://user-images.githubusercontent.com/70706948/221694202-42785765-8efc-4e44-a22c-7e3869e8f6b8.png)

**Comment by r15ch13** [comment]
Could you please post the output of:
```pwsh
 Get-Content "$env:USERPROFILE\scoop\shims\neofetch.cmd"
```
This will show the content of the `neofetch.cmd` shim file. I guess `á` in your username might be the problem. :smile:

**Reply by VictiniFan360** [reply]
> Could you please post the output of:
> 
> ```powershell
>  Get-Content "$env:USERPROFILE\scoop\shims\neofetch.cmd"
> ```
> 
> This will show the content of the `neofetch.cmd` shim file. I guess `á` in your username might be the problem. 😄

Hey, friend! Sorry for the late!
Here is the outpt:

![image](https://user-images.githubusercontent.com/70706948/223526670-d5acaa4f-3da7-41ab-8ae0-56d34ae5577e.png)
